app/plugins/modules/activation-visualizations/lib/util.js

```diff
@@ -64,13 +64,6 @@ const amendWithNamespace = name => {
     }
 }
 
-/**
- * The OpenWhisk name filter accepts only plain names, no package or
- * namespace, no slashes.
- *
- */
-const stripToPlainName = name => name.substring(name.lastIndexOf('/') + 1)
```

Comment:
The name may be either a package/action or action. A full name with the namespace is not supported.

Reply:
ok thanks for the clarification. i'll error out if the name query includes a namespace not equal to the currently selected namespace; if it includes a namespace equal to the current namespace, i'll strip that off, so that the query conforms